standalone/CMakeLists.txt

```diff
@@ -12,11 +12,15 @@ include(../cmake/CPM.cmake)
 
 CPMAddPackage(
   GITHUB_REPOSITORY jarro2783/cxxopts
-  VERSION 3.0.0
+  VERSION 3.1.1
   OPTIONS "CXXOPTS_BUILD_EXAMPLES NO" "CXXOPTS_BUILD_TESTS NO" "CXXOPTS_ENABLE_INSTALL YES"
 )
 
-CPMAddPackage(NAME Greeter SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/..)
+CPMAddPackage(
+  NAME Greeter
+  SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/..
+  FORCE ON
+)
```

Comment:
Can you shortly explain why `FORCE ON` is needed here?
If the Greeter project has already been added before I would expect it to use the previously added version?

Reply:
I think I understand, see review comment above.